src/dealDashboard/dealVotes/dealVotes.ts

```diff
@@ -1,36 +1,73 @@
-import { autoinject, bindable } from "aurelia-framework";
+import { autoinject, bindable, computedFrom } from "aurelia-framework";
 import { DealTokenSwap } from "entities/DealTokenSwap";
 import "./dealVotes.scss";
 import { Router } from "aurelia-router";
 import { EthereumService } from "../../services/EthereumService";
-import { AlertService } from "../../services/AlertService";
+import { FundingModal } from "./fundingModal/fundingModal";
+import { DialogService } from "../../services/DialogService";
 
 @autoinject
 export class DealVotes {
   @bindable deal: DealTokenSwap;
 
   fundingModal: HTMLElement;
 
+  everyTextCopy = [
+    {
+      condition: () => !this.deal.majorityHasVoted && !this.deal.representativeVoted(this.ethereumService.defaultAccountAddress),
+      voteText: "You’re eligible to vote. The deal will be approved once the majority accepts the deal",
+      statusText: "Voting in progress",
+    },
+    {
+      condition: () => !this.deal.majorityHasVoted && this.deal.representativeVoted(this.ethereumService.defaultAccountAddress) && this.deal.isTheProposalLead(this.ethereumService.defaultAccountAddress),
+      voteText: "Please wait for the representatives to cast their votes. The representatives are able to change their votes before the funding phase is initiated. If the deal is approved, you can initiate the funding phase",
+      statusText: "Voting in progress",
+    },
+    {
+      condition: () => !this.deal.majorityHasVoted && this.deal.representativeVoted(this.ethereumService.defaultAccountAddress) && !this.deal.isTheProposalLead(this.ethereumService.defaultAccountAddress),
```

Comment:
isVoteCastedProposalLead

Reply:
And here, could split up the first 2 bools

```ts
!this.deal.majorityHasVoted      // 1
&& this.deal.representativeVoted(this.ethereumService.defaultAccountAddress)  // 2
&& !this.deal.isTheProposalLead(this.ethereumService.defaultAccountAddress) // 3

// 1 + 2 = 4
isRepresentativeHasNotVoted

// 4 + 3
this.isRepresentativeHasNotVoted(..) && 3
```

and reuse 4 for `isVotingInProgressProposalLead` above